tests/test_data/test_loading.py

```diff
@@ -64,6 +81,15 @@ def setup_class(cls):
                 'segment': [3.0, 5.0],
                 'label': 'Rock climbing'
             }])
+        cls.proposal_results = dict(
+            frame_dir=cls.img_dir,
+            video_id='test_imgs',
+            total_frames=cls.total_frames,
+            filename_tmpl=cls.filename_tmpl,
+            out_props=[[['test_imgs',
+                         ExampleSSNInstance(1, 4, 10, 1, 1, 1)], 0],
+                       [['test_imgs',
+                         ExampleSSNInstance(2, 5, 10, 2, 1, 1)], 0]])
```

Comment:
Key `out_props` doesn't exist in test mode.

Reply:
It is better to create two types of inputs to simulate the forward in different modes.